public/lottiefiles/loading-linkta.json

```diff
@@ -0,0 +1 @@
"ty":"gr","bm":0,"hd":false,"nm":"","it":[{"ty":"sh","bm":0,"hd":false,"nm":"","d":1,"ks":{"a":0,"k":{"c":true,"i":[[0,0],[-3.0720000000000027,0],[-2.1119999999999948,-0.19200000000000728],[0,0],[-1.024000000000001,-1.2160000000000082],[0,0],[0,0],[0,0],[-2.6240000000000094,0.960000000000008],[0,0],[1.2800000000000011,1.4080000000000013],[0,0]],"o":[[3.584000000000003,-0.8319999999999936],[3.0719999999999885,0],[0,0],[2.495999999999995,0.960000000000008],[0,0],[0,0],[0,0],[0.8960000000000008,-1.2160000000000082],[0,0],[-2.240000000000009,-0.7680000000000007],[0,0],[0,0]],"v":[[80.648,74],[90.632,72.752],[98.408,73.04],[98.408,111.344],[103.688,114.608],[102.728,122],[80.648,122],[79.688,114.608],[84.968,111.344],[84.968,84.656],[79.688,81.392],[80.648,74]]}}},{"ty":"sh","bm":0,"hd":false,"nm":"","d":1,"ks":{"a":0,"k":{"c":true,"i":[[0,0],[0,4.799999999999997],[-5.311999999999998,0],[0,-4.864000000000004],[5.376000000000005,0]],"o":[[-5.311999999999998,0],[0,-4.864000000000004],[5.376000000000005,0],[0,4.799999999999997],[0,0]],"v":[[91.496,67.376],[83.528,60.176],[91.496,52.88],[99.56,60.176],[91.496,67.376]]}}},{"ty":"fl","bm":0,"hd":false,"nm":"","c":{"a":0,"k":[0.9725,0.5255,0.2235]},"r":1,"o":{"a":0,"k":100}},{"ty":"tr","a":{"a":0,"k":[0,0]},"s":{"a":1,"k":[{"o":{"x":0.34,"y":1.55},"i":{"x":0.65,"y":1},"s":[0,0],"t":6},{"s":[100,100],"t":60}]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.34,"y":1.55},"i":{"x":0.65,"y":1},"s":[175,87.5],"t":6},{"s":[0,0],"t":60}]},"r":{"a":0,"k":0},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.34,"y":1.55},"i":{"x":0.65,"y":1},"s":[0],"t":6},{"s":[100],"t":60}]}}]},{"ty":"gr","bm":0,"hd":false,"nm":"","it":[{"ty":"sh","bm":0,"hd":false,"nm":"","d":1,"ks":{"a":0,"k":{"c":true,"i":[[0,0],[0,0],[0,0],[-2.687999999999999,1.024000000000001],[0,0],[0.8960000000000008,1.216000000000001],[0,0],[0,0],[0,0],[2.495999999999995,-0.960000000000008],[0,0],[0,0],[-1.5360000000000014,1.4080000000000013],[0,0],[0,0]],"o":[[0,0],[0,0],[0.8320000000000007,-1.0879999999999939],[0,0],[-2.6239999999999988,-0.960000000000008],[0,0],[0,0],[0,0],[-1.024000000000001,1.216000000000001],[0,0],[0,0],[0.8320000000000007,-3.328000000000003],[0,0],[0,0],[0,0]],"v":[[70,122],[26.032,122],[25.072,114.032],[30.352,110.864],[30.352,66.992],[25.072,63.728],[26.032,55.76],[48.496,55.76],[49.456,63.728],[44.176,66.992],[44.176,111.536],[59.344,111.536],[62.896,104.432],[70,105.296],[70,122]]}}},{"ty":"fl","bm":0,"hd":false,"nm":"","c":{"a":0,"k":[0.9725,0.5255,0.2235]},"r":1,"o":{"a":0,"k":100}},{"ty":"tr","a":{"a":0,"k":[0,0]},"s":{"a":1,"k":[{"o":{"x":0.34,"y":1.55},"i":{"x":0.65,"y":1},"s":[0,0],"t":0},{"s":[100,100],"t":30}]},"sk":{"a":0,"k":0},"p":{"a":1,"k":[{"o":{"x":0.34,"y":1.55},"i":{"x":0.65,"y":1},"s":[175,87.5],"t":0},{"s":[0,0],"t":30}]},"r":{"a":0,"k":0},"sa":{"a":0,"k":0},"o":{"a":1,"k":[{"o":{"x":0.34,"y":1.55},"i":{"x":0.65,"y":1},"s":[0],"t":0},{"s":[100],"t":30}]}}]}],"ind":1}],"v":"5.7.0","fr":60,"op":190,"ip":0,"assets":[]}
```

Comment:
This looks good. We can create a backlog ticket to add a loader for the home page with this lottie.

Reply:
Thank you Kinski, your loader actually looks really good as well